src/binary-encoding.adoc

```diff
@@ -1096,27 +1217,49 @@ Unlike the short-form encoding, the long-form encoding reserves:
-`(coefficient, exponent)` pair, similar to a <<decimals, decimal>>. However, it is illegal for the fractional
-seconds value to be greater than or equal to `1.0` or less than `0.0`. For this reason, both the coefficient and
-the exponent are encoded using unsigned types. The included coefficient `FlexUInt` is unsigned to prevent the encoding of
-fractional seconds less than `0.0`. The exponent `FixedUInt` is implicitly negative, discouraging the encoding of
-decimal numbers greater than `1.0`. Note that validation is still required; namely:
+Similar to short-form timestamps, with the exception of representing the fractional seconds, the components of the
+timestamp are encoded as bit-fields on a `FixedUInt` that corresponds to the length that followed the opcode.
+
+If the timestamp's overall length is greater than or equal to `8`, the `FixedUInt` part of the timestamp is `8` bytes
+and the remainder of the length is used to encode fractional seconds. The fractional seconds are encoded as a
+`(coefficient, scale)` pair, which is _similar_ to a <<decimals, decimal>>. The primary difference is that the *scale*
+represents a negative *exponent* because it is illegal for the fractional seconds value to be greater than or equal to
+`1.0` or less than `0.0`. The coefficient is encoded as a `FlexUInt` (instead of `FlexInt`) to prevent the encoding of
+fractional seconds less than `0.0`. The scale is encoded as a `FixedUInt` (instead of `FixedInt`) to discourage the
+encoding of decimal numbers greater than `1.0`. Note that validation is still required; namely:
 
-* An exponent value of `0` is illegal, as that would result in a fractional seconds greater than `1.0` (a whole second).
-* If `coefficient * 10^-exponent > 1.0`, that `(coefficient, exponent)` pair is illegal.
+* A scale value of `0` is illegal, as that would result in a fractional seconds greater than `1.0` (a whole second).
+* If `coefficient * 10^-scale > 1.0`, that `(coefficient, scale)` pair is illegal.
 
 If the timestamp's length is `3`, the most significant bit in the final byte (`h`) is a flag
 that indicates month (`0`) or day (`1`) precision. If the timestamp's length is greater than `3`, the (`h`) bit is
 treated as the least-significant bit of the hour (`H`) bits.
 
-.Figure {counter:figure}: Encoding of the body of a long-form timestamp
-[source,%unbreakable]
-----
-     1         2         3         4         5         6         7         8           n
-+=========+=========+=========+=========+=========+=========+=========+========+   +=========+
-|YYYY:YYYY|MMYY:YYYY|hDDD:DDMM|mmmm:HHHH|oooo:oomm|ssoo:oooo|----|ssss|FlexUInt|...|FixedUInt|...
-+=========+=========+=========+=========+=========+=========+=========+========+   +=========+
+.Figure {counter:figure}: Encoding of the _body_ of a long-form timestamp
+[%unbreakable,source]
+----
+         +=========+
+byte 0   |YYYY:YYYY|
```

Comment:
In the other diagrams, byte 0 began with the opcode. Should we do that here too, for consistency?

Reply:
In this one, it was specifically only the body.  I had added #222 before this PR to cover this because when I went through this section I realized that we should have a bunch of examples w/opcodes.